### ISSUE TITLE
Sync State SQLLite

### ISSUE THREAD

**Issue by decanus** [status-im/status-console-client#119]
To ensure that the sync state does not get cleared every time we stop and restart the client, we need to store it in SQLLite rather than in memory.

**Comment by cammellos** [comment]
@decanus what about payloads, do they need to be persisted as well?

**Comment by decanus** [comment]
@cammellos, no. Only syncstate, message store and epoch.

**Comment by cammellos** [comment]
@decanus what about peers? is a client expected to re-add peers each time it starts up?

**Comment by decanus** [comment]
@cammellos yeah we should also persist those. Should I create something like a peerbook that would allow us to easier manage peers externally from the node/

**Comment by cammellos** [comment]
@decanus no worries, I have already added something on those lines here https://github.com/vacp2p/mvds/tree/bugfix/acks
